PhysicsTools/PatUtils/plugins/JetSubstructurePacker.cc

```diff
@@ -61,6 +61,28 @@ JetSubstructurePacker::produce(edm::Event& iEvent, const edm::EventSetup&)
       for ( auto const & jjet : *ialgoHandle ) {
 	
 	if ( reco::deltaR( ijet, jjet ) < dRMin ) {
+
+	  for ( auto const & userfloatstr : jjet.userFloatNames() ) {
```

Comment:
hi @rappoccio - doing this set of checks for each jet seems like an automatic performance killer especially as it is then repeated inside of addUserFloat.  How can these two checks be merged?

(ok, I see there are bigger performance fish in this PR..)

Reply:
OK, I can remove the check.